src/reducers/SignUp.ts

```diff
@@ -47,8 +47,6 @@ export const signUp = (state: any, action: any) => {
         loading: false,
       }
     default:
-      return {
-        defaultState,
-      }
+      return defaultState
```

Comment:
This will wipe out any state changes a previous reducer made.
default should *always* return 'state' in a reducer.

The reducer signature needs to be changed to:
```javascript
export const signUp = (state: any = defaultState, action: any) => {
}
```

Reducer unit tests would have caught this - please add some!